pkg/controller/volume/scheduling/scheduler_binder_test.go

```diff
@@ -1225,7 +1226,7 @@ func TestAssumePodVolumes(t *testing.T) {
 			scenario.expectedProvisionings = scenario.provisionedPVCs
 		}
 		if scenario.shouldFail {
-			testEnv.validateFailedAssume(t, pod, scenario.expectedBindings, scenario.expectedProvisionings)
+			testEnv.validateCacheRestored(t, pod, scenario.bindings, scenario.provisionedPVCs)
```

Comment:
We have to pass scenario.bindings and scenario.provisionedPVCs because scenario.expectedBindings, and scenario.expectedProvisionings are empty when scenario.shouldFail is true.  If these are empty, nothing is checked in validateFailedAssume.